### DISCUSSION TITLE
[Feature] Setting for max number of faces

### DISCUSSION THREAD

**Discussion by diyoyo** [immich-app/immich#5554] [Feature Request]
### The feature

Hi, i just notice the limit of 500 displayed faces in the People's Tab, and it keeps me wondering:
what happens to the other faces in the meantime?
Let me explain:
- the library was first loaded with images where many people were on the same picture, resulting in many faces being generated, sometimes just for 1 picture, sometimes a very small face in the back in the sade.
- Then the library had more professional, closeup pictures going in, and I am worried that these won't be properly clustered since they arrived later in time, when there was already more than 500 faces.

Everytime I hide a few useless faces, I can see more and more faces coming in.
Yet, I know for sure that the new faces should come in bigger clusters, not only from the bottom (when ranked by number of photos), but I can only detect single-pic people from the first group photos, not the closeup pics from the other set .

Not sure I express myself clearly enough, but at least, it could be nice in the UI to be able to load all faces, or more than 500, just to make sure we can do massive hiding, allowing important faces to show up.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
The limit is just visual and is due to browser performance limitations as that view is not lazy-loaded/paginated yet. This will be fixed in a future update. (#2472)

**Reply by diyoyo** [reply]
Ok, thanks. Also, I think it could be nice to have a toggle where you can automatically hide (default) face clusters with only 1 asset.
It makes it clear to the user that "Maybe it's not the right time for you to start naming faces, just wait until clustering is finished.".
If you see what I (a rather impatient person) mean 😉
